From affe53de99a23b8a2597e9210defaa4a206d6932 Mon Sep 17 00:00:00 2001
From: Joris Vaillant <joris.vaillant@inria.fr>
Date: Wed, 11 Jun 2025 11:18:36 +0200
Subject: [PATCH] [PATCH] Thin build on osx-64

---
 bindings/python/setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bindings/python/setup.py b/bindings/python/setup.py
index b98dfbef..873822fb 100755
--- a/bindings/python/setup.py
+++ b/bindings/python/setup.py
@@ -131,7 +131,8 @@ def build_libraries():
                   '-DCAPSTONE_BUILD_SHARED_LIBS=ON',
                   '-DCAPSTONE_BUILD_STATIC_LIBS=OFF',
                   '-DCAPSTONE_BUILD_LEGACY_TESTS=OFF',
-                  '-DCAPSTONE_BUILD_CSTOOL=OFF'
+                  '-DCAPSTONE_BUILD_CSTOOL=OFF',
+                  '-DCAPSTONE_BUILD_MACOS_THIN=ON'
                   ]
     cmake_build = ['cmake',
                    '--build',
-- 
2.49.0

