From 4969ae3db2af0d49271caffaa7e0a4741ff9b894 Mon Sep 17 00:00:00 2001
From: "Christopher J. Wright" <cjwright4242@gmail.com>
Date: Thu, 18 Mar 2021 17:16:27 -0400
Subject: [PATCH 1/3] skip IS_APPLE check on osx

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f84d43c1..4baa3a10 100644
--- a/Makefile
+++ b/Makefile
@@ -344,7 +344,7 @@ endif
 API_MAJOR=$(shell echo `grep -e CS_API_MAJOR include/capstone/capstone.h | grep -v = | awk '{print $$3}'` | awk '{print $$1}')
 VERSION_EXT =
 
-IS_APPLE := $(shell $(CC) -dM -E - < /dev/null 2> /dev/null | grep __apple_build_version__ | wc -l | tr -d " ")
+IS_APPLE := 1
 ifeq ($(IS_APPLE),1)
 # on MacOS, do not build in Universal format by default
 MACOS_UNIVERSAL ?= no
